src/components/families/family-form/student-form/StudentForm.tsx

```diff
@@ -152,7 +152,7 @@ const StudentForm = ({
                 )}
               </Box>
             )}
-            <Box flex="auto">
+            <Box flex="auto" width={isEditing && dense ? 224 : null}>
```

Comment:
Does it make sense to store 224 somewhere in a variable for readibility/context? the answer could very well be NO 😅

Reply:
added!